Release 1.13.0 (#4055)
redhat-developer/vscode-openshift-tools: CHANGELOG.md

# Change Log

## 1.13.0 (April 10, 2024)

>  * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.

Reviewer: Rename to "fix starting crc via the extension"

Reviewer: Acutally, use Mohit's suggestion

Author: Used Mohit's version for this: "Bump minimum memory for CRC to 10752 MB"

>  * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.

Reviewer 2: ```suggestion
 * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Bump minimum memory for CRC to 10752 MB
```

Author: ACK

>  * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Provide an option to delete the Pod like k8s.

Reviewer 2: ```suggestion
 * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Support deletion of Pod resources from extension UI.
```

Author: ACK

>  * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Survey doesn't work.

Reviewer: Rename to "fix survey"

Reviewer: Use Mohit's suggestion

Author: Used Mohit's version: `Fix Extension Survey form`

>  * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Survey doesn't work.

Author: ACK

>  * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Re-arrange application explorer similar to vsKubernetes cluster view.

Reviewer 2: ```suggestion
 * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Showcase all Workload resources in Application Explorer Tree view.
```

 * [#3981](https://github.com/redhat-developer/vscode-openshift-tools/issues/3981) Migrate Functionality from Kubernetes VSCode Extension to OpenShift Toolkit.

>  * [#3947](https://github.com/redhat-developer/vscode-openshift-tools/issues/3947) The App Explorer tree isn't refreshed after logging in to a Kind cluster.

Author: ACK

>  * [#3947](https://github.com/redhat-developer/vscode-openshift-tools/issues/3947) The App Explorer tree isn't refreshed after logging into a Kind cluster.

Author: ACK

 * [#3919](https://github.com/redhat-developer/vscode-openshift-tools/issues/3919) Improve "Open Kuberbetes YAML file" to avoid accidental opens and duplicate files.

>  * [#3918](https://github.com/redhat-developer/vscode-openshift-tools/issues/3918) Fix Action labels for kubernetes clusters.

Reviewer: I think this should be more specific, eg. "Use "namespaces" wording on kubernetes clusters and "projects" wording on OpenShift clusters"

Author: ACK
